providers/ms365/resources/ms365.lr.manifest.yaml

```diff
@@ -270,7 +271,27 @@ resources:
     fields:
       csOAuthConfiguration: {}
       csTeamsClientConfiguration: {}
+      csTeamsMeetingPolicy: {}
```

Comment:
if these new fields were all auto generated they should have a min mondoo version that lets us track these as being released in the 9.x cycle. That should get added to all these